server/webhook/merge_request.go

```diff
@@ -73,6 +73,8 @@ func (w *webhook) handleChannelMergeRequest(event *gitlab.MergeEvent) ([]*Handle
 		message = fmt.Sprintf("[%s] Merge request [#%v %s](%s) was merged by [%s](%s)", repo.PathWithNamespace, pr.IID, pr.Title, pr.URL, senderGitlabUsername, w.gitlabRetreiver.GetUserURL(senderGitlabUsername))
 	} else if pr.Action == "close" {
 		message = fmt.Sprintf("[%s] Merge request [#%v %s](%s) was closed by [%s](%s)", repo.PathWithNamespace, pr.IID, pr.Title, pr.URL, senderGitlabUsername, w.gitlabRetreiver.GetUserURL(senderGitlabUsername))
+	} else if pr.Action == "reopen" {
```

Comment:
Could we make this a bit cleaner with a switch statement?

Reply:
Sounds good to me. I thought doing this in the first commit, but I wasn't sure if the larger diff for a small change like this was a good idea.